src/sys/mod.rs

```diff
@@ -15,8 +15,8 @@ pub mod unix;
 
 #[cfg(windows)]
 pub use self::windows::{
-    event, Binding, Event, Events, Overlapped, Selector, TcpListener, TcpStream, UdpSocket, Waker,
+    event, Event, Events, Selector, SourceRawSocket, TcpListener, TcpStream, UdpSocket, Waker,
 };
 
 #[cfg(windows)]
-mod windows;
+pub mod windows;
```

Comment:
Small thing but I don't think this needs to be public, could be mistaken though.